packages/frontend/core/src/modules/doc-display-meta/services/doc-display-meta.ts

```diff
@@ -149,7 +152,7 @@ export class DocDisplayMetaService extends Service {
 
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line +145]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->should handle emoji icon<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/9165?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->